libcxx/test/libcxx/containers/associative/unord.set/abi.compile.pass.cpp

```diff
@@ -94,7 +92,7 @@ struct user_struct {
 };
 
 #if __SIZE_WIDTH__ == 64
-static_assert(sizeof(user_struct) == 48, "");
+static_assert(sizeof(user_struct) == 40, "");
```

Comment:
Throughout: please `#ifdef` out these tests that don't work on GCC -- actually those should keep checking against `48` bytes. Please also include a link to the GCC bug report for anonymous structs.